archivers/upx-devel/Portfile

```diff
@@ -0,0 +1,45 @@
+# $Id$
+
+PortSystem       1.0
+PortGroup        github 1.0
+
+github.setup     upx upx 5c0340f
+
+name             upx-devel
+version          20161019
+categories       archivers
+maintainers      openmaintainer
```

Comment:
We do not use openmaintainer on its own. If you want to maintain the port, add your domain.tld:localpart-formatted email address; otherwise, set this to nomaintainer.